docs/plugins/02-extra-plugins.md

```diff
@@ -351,6 +351,21 @@ lvim.builtin.treesitter.rainbow.enable = true
 
 ## Telescope Extensions
 
+### How to install telescope extensions
+
+First **add your telescope extension to the list of plugins** as usual (`lvim.plugins = { ... }`) following the extension instructions.
+There are several ways to register extensions within telescope, but the safer is using the `on_config_done` callback for telescope.
+Create the callback function anywhere in your `config.lua`. This function will be called when telescope has finished loading and will get telescope as its only parameter.
+Finally, within the `on_config_done` callback register your extension :
+
+```lua
+lvim.builtin.telescope.on_config_done = function(telescope)
+  telescope.load_extension "frecency"
+  telescope.load_extension "neoclip"
```

Comment:
```suggestion
  pcall(telescope.load_extension, "frecency")
  pcall(telescope.load_extension, "neoclip")
```
just so packer doesn't go haywire and try removing your plugins 😄 
otherwise you'd have a race-condition between packer trying to install the plugins for the first time, and packer trying to configure telescope.

you could also add some prints if you like, but personally I don't think the error message from these is that interesting, and running `:Telescope frecency` should tell you if it loaded correctly or not.